search-app/test/cmr/search/test/site/routes.clj

```diff
@@ -26,6 +25,21 @@
       (testing "returns the welcome page HTML"
         (is (substring? "The CMR Search API" (:body response)))))))
 
+(deftest landing-links-page
+  (testing "landing links page returns content"
+    (let [path "/site/collections/landing-pages"
+          response (site (request :get (str "https://cmr.example.com/search" path)))]
+      (is (= (:status response) 200))
+      (is (substring? "Collections Landing Pages" (:body response)))
+      (is (substring? "Landing Pages for EOSDIS Collections" (:body response))))))
+
+(deftest eosdis-landing-links-page
+  (testing "eosdis collections landing links page returns content"
+    (let [path "/site/collections/eosdis-landing-pages"
+          response (site (request :get (str "https://cmr.example.com/search" path)))]
+      (is (= (:status response) 200))
+      (is (substring? "Landing Pages for EOSDIS Collections" (:body response))))))
+
 (deftest test-404
   (testing "a 404 is returned for a missing document"
     (is (= 404
```

Comment:
As mentioned on chat: did a quick check, and it's in the file (see https://github.com/oubiwann/Common-Metadata-Repository/blob/c011bf4a47e548d0905858a5b442b647e2d62950/search-app/test/cmr/search/test/site/routes.clj); Github's UI isn't providing an expand-code icon here, so we're not able to see that there's more code.